ocs_ci/ocs/constants.py

```diff
@@ -602,8 +602,11 @@
 OCS_CSV_PREFIX = 'ocs-operator'
 LOCAL_STORAGE_CSV_PREFIX = 'local-storage-operator'
 LATEST_TAGS = (
-    'latest', 'latest-stable', '4.2-rc', 'latest-4.2', 'latest-stable-4.2',
+    'latest', 'latest-stable',
+    '4.2-rc', 'latest-4.2', 'latest-stable-4.2',
```

Comment:
do we need x.y-rc ? @andrewschoen

Reply:
I would keep it here for now if we will use them at some point as we were used to. It doesn't matter here, it's just used to identify such build and then ocs-ci will replace such tag with some specific.

Reply:
@petr-balogh if it has usage then its better to keep it but if latest-4.x will also point to rc then its better to get rid of this to avoid confusion and users setting this value and expecting rc build?